index.html

```diff
@@ -66,6 +66,170 @@ <h2 class="header-subtitle text-white font-inter">I’m a Software Developer</h2
+                <ul class="tags">
+                    <li class="tag">CSS</li>
+                    <li class="tag">HTML</li>
+                    <li class="tag">Bootstrap</li>
+                    <li class="tag">Ruby</li>
+                </ul>
+                <button class="button mt-12">See Project</button>
+            </div>
+            <div class="project-card">
+                <div class="project">
+                    <div>
+                        <h3 class="project-title text-white font-crete-round">Profesional Art Printing Data</h3>
+                        <p class="project-description text-white mt-12">A daily selection of privately personalized
+                            reads; no
+                            accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="tags">
+                            <li class="tag text-white dark-tag mb-6">HTML</li>
+                            <li class="tag text-white dark-tag mb-6">Bootstrap</li>
+                            <li class="tag text-white dark-tag mb-6">Ruby</li>
+                        </ul>
+                    </div>
+                </div>
+                <button class="button w-full">See Project</button>
+            </div>
+            <div class="project-card">
+                <div class="project">
+                    <div>
+                        <h3 class="project-title text-white font-crete-round">Profesional Art Printing Data</h3>
+                        <p class="project-description text-white mt-12">A daily selection of privately personalized
+                            reads; no
+                            accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="tags">
+                            <li class="tag text-white dark-tag mb-6">HTML</li>
+                            <li class="tag text-white dark-tag mb-6">Bootstrap</li>
+                            <li class="tag text-white dark-tag mb-6">Ruby</li>
+                        </ul>
+                    </div>
+                </div>
+                <button class="button w-full">See Project</button>
+            </div>
+            <div class="project-card">
+                <div class="project">
+                    <div>
+                        <h3 class="project-title text-white font-crete-round">Profesional Art Printing Data</h3>
+                        <p class="project-description text-white mt-12">A daily selection of privately personalized
+                            reads; no
+                            accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="tags">
+                            <li class="tag text-white dark-tag mb-6">HTML</li>
+                            <li class="tag text-white dark-tag mb-6">Bootstrap</li>
+                            <li class="tag text-white dark-tag mb-6">Ruby</li>
+                        </ul>
```

Comment:
- [ ] [OPTIONAL] These technologies tags are done well. although you should remember to use the correct font from the design I notice they don't have the correct font set.

![image](https://user-images.githubusercontent.com/18730106/153671763-056a7010-ec7c-486e-99f4-e749c07e7e1b.png)

![image](https://user-images.githubusercontent.com/18730106/153671775-36bf50ba-7e21-445c-b75f-22d3940b0690.png)

Reply:
Thanks @MrBrN197, I fixed the font issue as well.